user/languages/ruby.md

```diff
@@ -248,6 +248,25 @@ versions](https://github.com/acrmp/chefspec/blob/master/.travis.yml):
 The same technique is often applied to test against multiple databases, templating
 engines, hosted service providers and so on.
 
+### `$BUNLDE_GEMFILE` environment variable
+
+When `gemfile` is thus defined *and* the file exists in the repository,
+we define the environment variable `$BUNLDE_GEMFILE`, so that `bundle install`
+respects this.
+
```

Comment:
I try to avoid using 'thus'. In this case, you're saying if `gemfile` is defined *and* exists we set $GEMFILE to that value, right?